src/dev/src/app/combobox/basic-ng-combobox.html

```diff
@@ -9,10 +9,10 @@
   Select 2.0 implementation is still incomplete. DO NOT USE.
   All issues related to clr-select will be closed until the implementation is completed.
 -->
-<clr-combobox>
```

Comment:
Reason?

Reply:
AOT errors since combobox is not actually a public in clr-angular yet